js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js

```diff
@@ -575,17 +575,17 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
         return false;
     },
 
-    _isRtlMode: function() {
-        return this._$parentContainer && this._$parentContainer.parent().css('direction') === 'rtl';
+    _isRtlParentStyle: function() {
+        return this._$parentContainer && this.option('rtlEnabled') && this._$parentContainer.parent().css('direction') === 'rtl';
```

Comment:
You can minify this condition using optional chaining. 
`this.option('rtlEnabled') && this._$parentContainer?.parent.().css.('direction') === 'rtl';`